microsoft-edge/devtools-guide-chromium/remote-debugging/surface-duo-emulator.md

```diff
@@ -0,0 +1,99 @@
+
+Launch the [Surface Duo emulator][DuoEmulator].
+
+> ##### Figure 2
+> The Surface Duo emulator
+> ![The Surface Duo emulator][ImageDuoEmulator]  
+
+## Step 3: Load your web content in Microsoft Edge on the Surface Duo emulator
+
+Drag the Favorites Tray up on either screen of the [Surface Duo emulator][DuoEmulator] to pull up the Apps Drawer. Click **Edge** to launch the [Microsoft Edge app][AndroidEdge] app on the emulator.
+
+> ##### Figure 3
+> The Microsoft Edge app on the Surface Duo emulator
+> ![The Microsoft Edge app on the Surface Duo emulator][ImageDuoEmulatorEdge]  
+
+Navigate to the website or app that you want to debug in the [Microsoft Edge app][AndroidEdge].
+
+## Step 4: Debug your web content from the Surface Duo emulator in desktop Microsoft Edge
+
+Switch back to the desktop instance of [Microsoft Edge][DesktopEdge]. The `edge://inspect` page now shows the **SurfaceDuoEmulator** with a list of the tabs or [PWAs][PwaDocs] that are running on the [Surface Duo emulator][DuoEmulator].
+
+> ##### Figure 4
+> The `edge://inspect` page displays a list of the tabs in the Microsoft Edge app on the emulator
+> ![The edge://inspect page displays a list of the tabs in the Microsoft Edge app on the emulator][ImageEdgeInspectTargets]  
+
+> [!NOTE]
+> If you do not see **SurfaceDuoEmulator** on the `edge://inspect` page, try opening or closing tabs in the [Microsoft Edge app][AndroidEdge] on the [Surface Duo Emulator][DuoEmulator]. Refer to our [troubleshooting section for Android devices][TroubleshootingAndroid] if that does not resolve the issue.
+
+Click **inspect** for the web content you want to debug. The [Microsoft Edge DevTools][DevToolsDocs] will open in a new window. Click **Toggle Screencast** ![Toggle Screencast][ImageToggleScreencastIcon] to view the content on your [Surface Duo emulator][DuoEmulator] in your DevTools window. You are now able to use the full power of the Microsoft Edge DevTools to debug your web content running on the [Surface Duo emulator][DuoEmulator].
+
+> ##### Figure 5
+> Using the Microsoft Edge DevTools against Bing in the Microsoft Edge app on the Surface Duo emulator
+> ![Using the Microsoft Edge DevTools against Bing in the Microsoft Edge app on the Surface Duo emulator][ImageDevTools]  
+
+> [!NOTE]
+> If you span the [Microsoft Edge app][AndroidEdge] across both screens in the emulator, the screencast will reflect the new size of the app but not the hinge. To understand how the hinge impacts the layout of your web content, use the [Surface Duo emulator][DuoEmulator] instead of the screencast.
+
+## Additional Resources
+
+The web is a great platform for the new class of foldable and dual-screen devices because you can write your HTML, CSS, and JavaScript once and have it look great across single-screen, dual-screen, and foldable devices. Check out the additional resources below to get started building great web content for these new devices:
```

Comment:
```suggestion
The web is a great platform for the new class of foldable and dual-screen devices because you can write your HTML, CSS, and JavaScript once and have it look great across single-screen, dual-screen, and foldable devices. For more information, see these additional resources to get started building web content for these new devices.
```